README.md

```diff
@@ -70,12 +86,10 @@ const state = proxy({ count: 0 })
 
 function App() {
   return (
-    <div>
-      {state.count} // This is not recommended as it is not reactive.
-    </div>
+    <div>{state.count} // This is not recommended as it is not reactive.</div>
   )
 }
-``` 
+```
 
 ### Snapshots mutating is not possible
```

Comment:
Add h2 tag? What's the appropriate label?

```suggestion
## ...

### Snapshots mutating is not possible
```

Reply:
Ah, why would I add one here? isn't this the part of the `why` 

_Validations_, _Dos and Don'ts_ , _Rules_ ?